frontend/src/app/components/dashboard/db-table-view/db-table-row-view/db-table-row-view.component.css

```
@@ -30,13 +30,72 @@
+		background: rgba(0, 0, 0, 0.2);
+		margin: 8px auto 0;
+	}
+}
+
+@media (prefers-color-scheme: dark) {
+	.row-preview-sidebar__handle {
+		background: rgba(255, 255, 255, 0.24);
+	}
+}
+
+@keyframes rowPreviewSlideUp {
```

Stylelint (17.12.0)

[error] Expected keyframe name "rowPreviewSlideUp" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)